src/main.c

```diff
@@ -6,7 +6,8 @@
 
 #include <stdlib.h>
 #include <stdint.h>
-#include <emmintrin.h>
+#define SIMDE_ENABLE_NATIVE_ALIASES
```

Comment:
This one's also missing an undef.

Reply:
Looks like no intrinsics are used in this file, so I've removed the #define and the include